FirebaseStorage/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.0.1
+- [fixed] When passing in metadata to `putFile`, the metadata is properly initialized (#10353)
```

Comment:
nit: Phrase this in past tense and try to avoid passive voice.

```
Fixed an issue where metadata passed to `putFile` was not properly initialized.
```